pkg/controller/replicaset/replica_set_test.go

```diff
@@ -357,15 +359,15 @@ func TestStatusUpdatesWithoutReplicasChange(t *testing.T) {
 
 	// This response body is just so we don't err out decoding the http response, all
 	// we care about is the request body sent below.
-	response := runtime.EncodeOrDie(testapi.Extensions.Codec(), &extensions.ReplicaSet{})
```

Comment:
`testapi.Apps` defaults to `apps/v1beta1`. This can only be changed if we change https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/apps/install/install.go#L40. But doing so changes the default storage version.